utils/countries.tsx

```diff
@@ -0,0 +1,45 @@
+import countriesJSON from '../src/assets/countries.json';
+import parsePhoneNumber from 'libphonenumber-js';
+
+const countries: {
+    [key: string]: {
+        name: string;
+        native: string;
+        currency: string;
+        languages: string[];
+        emoji: string;
+    };
+} = countriesJSON;
+
+export const countriesOptions = Object.entries(countries).map(([key, value]: any) => ({
+    label: value.name,
+    value: key
+}));
+
+export function getCountryFromPhoneNumber(pnumber: string) {
+    const phoneNumber = parsePhoneNumber(pnumber);
+
+    if (phoneNumber?.country) {
+        const { emoji, name } = countries[phoneNumber.country];
+
+        return `${emoji} ${name}`;
+    }
+
+    return 'Unknown';
+}
```

Comment:
You won't need this, neither the import `import parsePhoneNumber from 'libphonenumber-js';` Please remove.